docs/source/configuration/key_value_service_configs/oracle_key_value_service_config.rst

```diff
@@ -19,7 +19,7 @@ e.g.
 
 .. code-block:: groovy
 
-  runtime 'com.palantir.atlasdb:atlasdb-dbkvs:0.19.0'
+  runtime 'com.palantir.atlasdb:atlasdb-dbkvs:0.25.0'
```

Comment:
It's possible to macro code to insert the latest release here, rather than manually updating it every so often.

@harrybiddle probably knows how.